DNN Platform/Modules/DnnExportImport/Components/Controllers/BaseController.cs

```diff
@@ -38,6 +41,13 @@ static BaseController()
             }
         }
 
+        public BaseController(IEnumerable<BasePortableService> portableServices)

```

Comment:
Do we need to adjust the parameterless constructor to make it obsolete and resolve BaseProtableServices ?